cachito/web/config.py

```diff
@@ -21,14 +22,17 @@ class DevelopmentConfig(Config):
     SQLALCHEMY_DATABASE_URI = 'postgresql+psycopg2://cachito:cachito@db:5432/cachito'
     SQLALCHEMY_TRACK_MODIFICATIONS = True
     CACHITO_SHARED_DIR = os.path.join(tempfile.gettempdir(), 'cachito-shared')
+    LOGIN_DISABLED = True
 
 
 class TestingConfig(DevelopmentConfig):
     """The testing Cachito Flask configuration."""
+    CACHITO_WORKER_USERNAMES = ['worker']
     # IMPORTANT: don't use in-memory sqlite. Alembic migrations will create a new
     # connection producing a new instance of the database which is deleted immediately
     # after the migration completes...
     #   https://github.com/miguelgrinberg/Flask-Migrate/issues/153
     SQLALCHEMY_DATABASE_URI = f'sqlite:///{TEST_DB_FILE}'
     DEBUG = True
+    LOGIN_DISABLED = False
```

Comment:
Consider defining this in the `Config` class so it's always defined.

Reply:
This is just here because the `TestingConfig` class inherits from the `DevelopmentConfig`. It defaults to `False`.